.github/workflows/sync_docs.yml

```diff
@@ -49,17 +49,15 @@ jobs:
           path: pantsbuild.org
 
       # Calculate destination
-      - name: Calculate branch name
-        id: get-branch-name
-        run: echo "BRANCH_NAME=$(echo "${{ inputs.version }}" | sed -E 's/^([0-9]+)\.([0-9]+)\.\w+.*/\1.\2.x/')" >> "${GITHUB_OUTPUT}"
       - name: Calculate destination
         id: get-destination-dir
         run: |
-          DESTINATION_DIR="versioned_docs/version-${{ steps.get-branch-name.outputs.BRANCH_NAME }}"
+          BASE_VERSION=$(echo "${{ inputs.version }}" | sed -E 's/^([0-9]+)\.([0-9]+)\.\w+.*/\1.\2/')
+          DESTINATION_DIR="versioned_docs/version-$BASE_VERSION"
           if [ ! -d "pantsbuild.org/$DESTINATION_DIR" ]; then
             DESTINATION_DIR="docs"
           fi
-          echo "DESTINATION_DIR=$(echo "$DESTINATION_DIR")" >> "${GITHUB_OUTPUT}"
+          echo "DESTINATION_DIR=$(echo "$DESTINATION_DIR")" | tee -a "${GITHUB_OUTPUT}"
```

Comment:
Drive-by change, for non-confidential values like this, printing the value to stdout helps with debugging, and `tee -a` is a handy way to do so without needing to repeat the `echo`s.

Reply:
I'm gonna try really hard to keep this one in my pocket.